drivers/sensor/bosch/bme280/bme280_async.c

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2024 Intel Corporation
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/logging/log.h>
+
+#include "bme280.h"
+
+LOG_MODULE_DECLARE(BME280, CONFIG_SENSOR_LOG_LEVEL);
+
+int bme280_submit(const struct device *dev, struct rtio_iodev_sqe *iodev_sqe)
```

Comment:
This should check and validate that the channels to read are correct

See how the channel array can be obtained here https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/sensor/tdk/icm42688/icm42688_rtio.c#L49

Reply:
Updated with a check